tests/serializers/lease/test_lease_serializer.py

```diff
@@ -13,8 +13,7 @@ def test_serializer(self, create_lease):
                 'name': lease.name,
                 'dateTimeStart': Time.format_date(lease.dateTimeStart),
                 'dateTimeEnd': Time.format_date(lease.dateTimeEnd),
-                'dateUpdated': Time.format_date(lease.dateUpdated),
+                'updated_at': None,
```

Comment:
I'm not sure if we will always want to keep updated_at defaulted to None.
None is technically an implementation detail. Perhaps we can do this instead:
```suggestion
                'updated_at': Time.serialized_date_format(lease.updated_at),
```

Reply:
> Very nice! Thanks for doing this. A couple small suggestions and questions.
> 
> It looks like `serialized_time_format`, the default db structure, will be used for sending date/time to the frontend. And `time_format` will be used for deserializing the date/time received from the frontend?

Not quite, that serialized_date_format method is just for the serializer tests that are doing a "dump" of the object from the db, while our resources are using the object model's json() method to explicitly send back the different properties to the frontend. Ex from issue 390 on the frontend: (emergency contacts)
![image](https://user-images.githubusercontent.com/8600792/96541743-7e92c480-1255-11eb-9abb-6aec5d329c55.png)

Reply:
As of yet the serializer is not hooked up to anything. Prior to this PR the serializer behavior matched the json() behavior. However, this PR is suggesting we change the behavior, and I'm trying to understand why.

From what your saying, It sounds like the serializer behavior should be reverted back to match the behavior of the json() method, because, regardless if we use the serializer module or not, the "dump" of the object from the db is what the FE will be receiving.

The output of the serializer should match the output of the json(); otherwise it is going to cause confusion going forward.

Reply:
This PR isn't changing the behavior of the serializer, it's updating the tests to match the current behavior

Reply:
> This PR isn't changing the behavior of the serializer,

If the behavior was not changed then the tests would not need to be changed.

The deleted lines containing the `created_at` and `updated_at` fields in the schema files changed the behavior.
<img width="460" alt="Screen Shot 2020-10-21 at 2 18 17 PM" src="https://user-images.githubusercontent.com/19519317/96676561-9038b100-13a8-11eb-9a2c-8afa26ca2ecd.png">

Those lines specify which format to use when dumping the data. Was there another reason to delete those lines, if not to change the behavior?

Reply:
Ahh okay, I was under the impression they were repetitive from the BaseModel already containing them. I can put them back! Or is there a way to just define the format in the BaseModel for all classes?

Reply:
The Model's define the schema at the database level. These schemas are for marshmallow and define how we dump/load and validate data. As far as I know there is not a way to define that at the database level. That is not really the job of the database.

Reply:
Interesting. I'll fix this tomorrow!